support_me/settings.py

```diff
@@ -148,3 +158,6 @@
 LOGIN_REDIRECT_URL = '//'
 
 SITE_ID = 2
+
+#heroku
+django_heroku.settings(locals())
```

Comment:
Should we add debug = FALSE if on heroku?
I've only done that by making heroku settings another file... is there a way to do it directly in this settings file?

Reply:
I added heroku_settings.py. Good catch!